tests/e2e/Pages/MapComponent.ts

```diff
@@ -46,6 +47,9 @@ class MapComponent extends DashboardPage {
     this.layerMenuToggle = this.page.getByTestId('layer-menu-toggle-button');
     this.redCrossMarker = this.page.getByAltText('red-cross-branch-marker');
     this.gloFASMarker = this.page.getByAltText('glofas-station-marker');
+    this.alerThresholdLines = this.page.locator(
+      '[stroke="var(--ion-color-ibf-outline-red)"]',
```

Comment:
is this safe enough? What is 'stroke' here exactly? Can there be nothing else that ends up in this filter?

Reply:
This is not ideal but after careful consideration this was the perfect and only candidate. That is why I kept it. It is also safe enough since it only appears on this particular element. Time will tell with other tests if it has to be updated. For now to save the dev time it is the only option.